addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2734,7 +2734,7 @@ const userValueWidgetsRegistry = {
 const SnippetOptionWidget = Widget.extend({
     tagName: 'we-customizeblock-option',
     events: {
-        'click .o_we_collapse_toggler': '_onCollapseTogglerClick',
+        'click .o_we_collapse_toggler, we-collapse .o_we_clickable': '_onCollapseTogglerClick',
```

Comment:
I think Oussama already tackled that problem for its gray configuration widget in the third panel's tab. I think we did not add a new class but simply reused the `o_we_collapse_toggler` on the element you want to be a toggler (and the class does come with CSS rules in special cases only).

Reply:
:+1: just a small detail: no "cursor:pointer" style on toggler (not sure if it's required here).

Reply:
Ok then maybe the class should at least force `cursor: pointer` indeed :+1: